curp/src/server/raw_curp/tests.rs

```diff
@@ -29,6 +29,13 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
         self.st.read().role
     }
 
+    fn contains(&self, id: ServerId) -> bool {
```

Comment:
What is the semantics of this method? To check whether the given node is the current member of this cluster? If so, why can not directly check via `cluster.get_members`?

Reply:
This is a test utility to confirm that the node modifies all the parts that need to be modified.